node/src/types/status_feed.rs

```diff
@@ -74,6 +76,17 @@ impl ChainspecInfo {
     }
 }
 
+/// The various possible states of operation for the node.
+#[derive(Clone, Copy, DataSize, Debug, Deserialize, Serialize, JsonSchema)]
+pub enum NodeState {
+    /// The node is currently in the fast syncing state.
+    FastSyncing,
+    /// The node is currently in the archival syncing state.
+    ArchivalSyncing,
+    /// The node is currently in the participating state.
+    Participating,
+}
+
```

Comment:
As soon as [this PR](https://github.com/casper-network/casper-node/pull/2704) is merged the names of the processes will be changed. Please see the description of the mentioned PR.

Reply:
Renamed as requested